sky/backends/cloud_vm_ray_backend.py

```diff
@@ -508,6 +508,9 @@ def _update_blocklist_on_gcp_error(self, region, zones, stdout, stderr):
                     # However, UNSUPPORTED_OPERATION is observed empirically when VM is preempted during creation.
                     # This seems to be not documented by GCP.
                     self._blocked_zones.add(zone.name)
+                elif code in ['RESOURCE_NOT_READY']:
+                    # This code is returned when the VM is still STOPPING.
+                    self._blocked_zones.add(zone.name)
```

Comment:
Why do we block the zone in this case?  Is it possible that
- a VM is STOPPING
- a concurrent `sky start` somehow hits this branch

From users' perspectives, the `sky start` should start the same VM in its original zone?

Reply:
As explained offline, failover does not apply here. so `sky start` would just fail if this error code is returned.